cosmwasm/deployment/terra2/tools/deploy.js

```diff
@@ -251,7 +240,19 @@ for (const [contract, registrations] of Object.entries(
         memo: "",
       })
       .then((tx) => terra.tx.broadcast(tx))
-      .then((rs) => console.log(rs));
+      .then((rs) => console.log(rs))
+      .catch((error) => {
+        if (error.response) {
+          // Request made and server responded
+          console.log(error.response.data, error.response.status, error.response.headers);
```

Comment:
console.log prints to stdout, since the messages are errors, we should log to stdere with console.error